frontend/src/metabase/common/components/EllipsifiedPath/EllipsifiedPath.module.css

```diff
@@ -0,0 +1,27 @@
+.path {
+  flex-grow: 1;
+  display: flex;
+  flex-flow: row nowrap;
+  overflow-x: hidden;
+}
+
+.pathSeparator {
+  opacity: 0.5;
+  min-width: 0;
+  flex-shrink: 0;
+  padding-inline: 0.1rem;
+}
+
+.pathItem {
+  flex-grow: 1;
+  min-width: 0;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
i'm not seeing the ellipsis actually happening here.

![Screen Shot 2024-08-06 at 10 19 50 AM](https://github.com/user-attachments/assets/58a8a615-5285-4a4a-be71-3f6e68a0217d)